.hashibot.hcl

```diff
@@ -7,8 +7,36 @@ behavior "remove_labels_on_reply" "remove_stale" {
     labels = ["waiting-reply"]
 }
 
+poll "stale_issue_closer" "waiting_reply_closer" {
+    schedule = "1 0 * * *"
+    no_reply_in_last = "480h" # 20 days
+    max_issues = 50
```

Comment:
You can crank this up to get through the backlog faster

Reply:
We try and go slow as not to spam everyone with emails 😬